src/recovery/mod.rs

```diff
@@ -64,7 +64,7 @@ const INITIAL_WINDOW: usize = INITIAL_WINDOW_PACKETS * MAX_DATAGRAM_SIZE;
 
 const MINIMUM_WINDOW: usize = 2 * MAX_DATAGRAM_SIZE;
 
-const MAX_DATAGRAM_SIZE: usize = 1452;
+pub const MAX_DATAGRAM_SIZE: usize = 1452;
```

Comment:
Once `INITIAL_WINDOW` and `MINIMUM_WINDOW` are removed, this should be moved to `lib.rs` instead.

Reply:
Moved.